apps/pie-docs/test/visual/navigation.spec.js

```diff
@@ -26,7 +26,7 @@ test.describe('PIE - Site Nav Menu', () => {
         await disableCookieBanner(page, context, true);
     });
 
-    test('Should open and close the mobile navigation menu - @mobile', async ({ page }) => {
+    test('Should open and close the navigation menu', async ({ page }) => {
```

Comment:
Sorry, I read this test again so wanted to check, doesn't the mobile nav only open on small screens (mobile and tablet) anyway? So adding desktop probably doesn't really test anything visually unless i am missing something here?